syntax = "proto3";

package company_token;

message CompanyToken {
    string id = 1;
    string company_id = 2;
    string token = 3;
    string created_at = 4;
    string expired_at = 5;
}